.github/crates-filters.yml

```diff
@@ -1,23 +1,3 @@
-consensus-config:
```

Comment:
Where are the consensus crates now?
Don't we need them as well?

Reply:
Re-added!